Jenkinsfile-Release

```diff
@@ -46,12 +31,11 @@ properties([
 ])
 
 final chartVersion = calculateChartVersion(params.chartVersion, params.appVersion)
+final nexusRepoManagerArtifactName = 'nexus-repository-manager-' + params.appVersion + '.tgz'
+final awsResiliencyArtifactName = 'nxrm-aws-resiliency-' + + params.appVersion + '.tgz'
```

Comment:
I feel like "params.chartVersion" is likely a better parameter to use for building these artifact names. Also, you can use the groovy string expression language for this: `"nexus-repository-manager-${params.chartVersion}.tgz"`

Reply:
Changed to params.chartVersion for both

Reply:
Per my current orservation the chartVersion is different between 'nxrm-aws-resiliency' and 'nexus-repository-manager' but appVersion is the same